src/components/POLine/Location/LocationForm.js

```diff
@@ -23,13 +23,21 @@ const reduceLocations = (locations, propName) => {
   return locations.reduce(reducer, 0);
 };
 
+const validateShouldBePositive = (value) => {
```

Comment:
you're still have misleading name of function (`positive` in it). `0` is neither negative or positive. So maybe it's better to name it as `validateNotNegative`, what do you think?

Reply:
fixed